templates/base.html

```diff
@@ -84,18 +85,31 @@
   <script src="{% static 'js/select2.min.js' %}"></script>
   <script type="text/javascript">
   $(document).ready(function() {
+    // Get language code in url, prepend to array, and remove duplicate
+    dataArray = JSON.parse('{{request.LANGUAGE_CODE|create_select2_data|safe}}')
+
+    // Populate select2
     $('.lang-select-box').select2({
-        data: [
-            {
-                id: 'en',
-                text: 'English'
-            },
-            {
-                id: 'fr',
-                text: 'French'
-            }
-        ]
+        data: dataArray,
+        width: '100px'
     });
+
+    // Attach click event to dropdown, which reloads page with new language code 
+    transObj = {
```

Comment:
Since we use this translation mapper object twice, and I imagine that we might want to edit it in the future, how about we move it to a global variable (like `AVAILABLE_TRANSLATIONS` or something similar) in the settings file and pass it into the view?

Reply:
The objects are slightly different - one converts the abbreviation to a full word ('fr' to 'French'), and the other converts the full word to an abbreviation ('French' to 'fr'). I think could consolidate them into one object, e.g., `translation_lookups`...I'll do that and put it in the settings!

Reply:
Ooooh OK I see! I missed that this one was performing the opposite lookup.

Reply:
...and on second thought, we'll need to set-up a custom context processor to give base.html access to the settings variables, and I am not sure it's worth the effort.

Reply:
Fine by me! Maybe if we find ourselves defining the available translations in any more places, we can decide to write the context processor then.